<?xml version="1.0"?>
<launch>
  <node name="serial_comm_read_node" pkg="serial_comm" type="comm_read" output="screen" >
        <param name="serial_port_name" value="/dev/ttyUSB1" />
        <param name="baudrate" value="B9600" />
        <param name="Hz" value="50.0" />
        <!-- <remap from="cmd_vel" to="cmd_vel"/> -->
  </node>
</launch>
